tests/integration_test.py

```diff
@@ -237,6 +242,19 @@ def test_drop_glyph_names(self, testufo, output_format, options, expected_ttx):
         ttf = compile_func(testufo, **options)
         expectTTX(ttf, expected_ttx)
 
+    def test_Instructions(self, instructions_ufo):
+        ttf = compileTTF(
+            instructions_ufo,
+            convertCubics=False,
+            reverseDirection=False,
+            removeOverlaps=False
+        )
+        assert "cvt " not in ttf
+        assert "gasp" in ttf
+        assert "fpgm" not in ttf
```

Comment:
won't these asserts have to change after we merge the PR with hinting support?

Reply:
Yes, this is a test I adapted from #335. The asserts must be inverted there.